releasenotes/notes/add-control-flow-to-consolidate-blocks-e013e28007170377.yaml

```
@@ -0,0 +1,9 @@
+---
+features:
+  - |
+    Enabled performing the :class:`qiskit.transpiler.passes.ConsolidateBlocks` pass inside the
+    blocks of :class:`qiskit.circuit.ControlFlowOp`. This pass collects several sequences of gates
+    and replaces each sequence with the equivalent numeric unitary gate. This new feature enables
+    applying this pass recursively to the blocks in control flow operations. Note that the meaning
+    of "block" in `:class:`qiskit.transpiler.passes.ConsolidateBlocks` is unrelated to that in
+    :class:`qiskit.circuit.ControlFlowOp`.
```

```suggestion
    Enabled performing the :class:`.ConsolidateBlocks` pass inside the
    blocks of :class:`.ControlFlowOp`. This pass collects several sequences of gates
    and replaces each sequence with the equivalent numeric unitary gate. This new feature enables
    applying this pass recursively to the blocks in control flow operations. Note that the meaning
    of "block" in :class:`.ConsolidateBlocks` is unrelated to that in
    :class:`.ControlFlowOp`.
```

The prefixed `.` means that Sphinx will do a suffix lookup on the entire database, and display only what comes after the dot.  Putting in the full path means that Sphinx will put in the whole text, which makes it hard to read for users.

---

I used other release notes as examples. But this is an improvement.